assets/in

```diff
@@ -72,7 +73,15 @@ for branch in $fetch; do
   git branch $branch FETCH_HEAD
 done
 
+return_ref=""
+if [ -n "$tag_filter" ]; then
```

Comment:
how is tag_filter used to match the tags? it looks like it is not being used at all.